llvm/test/CodeGen/SystemZ/builtin-setjmp-longjmp-02.ll

```diff
@@ -0,0 +1,89 @@
+;Test  longjmp load from jmp_buf.
```

Comment:
What is the added benefit of this combined test?   This seems to have been compiled from a C language test that would be useful to run as *execute* test (which we cannot do here).   But here we only want to verify `llc` code-gen, and it seems to me everything this test does was already separately verified by the setjmp and longjmp test cases.

Reply:
We certainly do not need to verify e.g. printf codegen here, so even if we do need the test for some reason, it should be significantly simplified.